src/osal/osal_freertos.h

```diff
@@ -32,6 +32,7 @@
 #include "semphr.h"
 #include "queue.h"
 #include "task.h"
+#include "tusb_option.h"
```

Comment:
this shouldn't be needed, since osal_freertos.h is only meant to be included by osal.h which already included tusb_common.h.

Reply:
agreed, but I had a couple of failed builds and opted for "better safe than sorry" approach :)